package.json

```diff
@@ -63,19 +62,29 @@
     "eslint-plugin-mocha": "6.x",
     "eslint-plugin-react": "7.x",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "history": "^5.3.0",
     "history-v4": "npm:history@^4.9",
-    "history-v5": "npm:history@^5.0",
     "jest": "^29.0.3",
     "jest-environment-jsdom": "^29.0.3",
     "jest-fetch-mock": "^3.0.3",
-    "prettier-eslint-cli": "^5.x",
+    "prettier-eslint-cli": "^7.1.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "ts-node": "^10.9.1",
     "typescript": "^4.8.3"
   },
   "peerDependencies": {
+    "@types/history": "^4.9",
+    "history": "^4.9 || ^5.0.0",
```

Comment:
**The only way** that course-search-utils uses `history` is as a type annotation for the `useCourseSearch` export, which has a history object as one of its argument. **This library does use history directly.**

`history` should be a peer dependency because we want to be sure that course-search-utils and the parent app are using the same version of history. Additionally, it's an **optional** peer dependency because: if you're not using the `useCourseSearch` export, you don't need history in your app.

`@types/history` is optional for the same reason. *Additionally, you don't need `@types/history` if you're not using typescript, or if you're using history v5, which includes its own types.*